docs/static/persistent-queues.asciidoc

```diff
@@ -53,44 +51,46 @@ process:
 input → persistent queue → filter + output 
 
 The input stage reads data from the configured data source and writes events to
-the persistent queue for processing. As events pass through the pipeline,
-Logstash pulls a batch of events from the persistent queue for processing them
-in the filter and output stages. As events are processed, Logstash uses a
-checkpoint file to track which events are successfully acknowledged (ACKed) as
-processed by Logstash. An event is recorded as ACKed in the checkpoint file if
-the event is successfully sent to the last output stage in the pipeline;
-Logstash does not wait for the output to acknowledge delivery. 
-
-During a normal, controlled shutdown (*CTRL+C*), Logstash finishes
+the persistent queue for processing. Logstash reads a batch of events from the
+persistent queue and sends them to the filter and output stages. As events
+are processed, Logstash records a checkpoint to track which events are
+successfully acknowledged (ACKed) as processed by Logstash. An event is
+recorded as ACKed in the checkpoint if the event is successfully sent to the
+last output stage in the pipeline;
+
+During a normal, controlled shutdown (*CTRL+C* or SIGTERM), Logstash finishes
 processing the current in-flight events (that is, the events being processed by
 the filter and output stages, not the queued events), finalizes the ACKing
 of these events, and then terminates the Logstash process. Upon restart,
-Logstash uses the checkpoint file to pick up where it left off in the persistent
-queue and processes the events in the backlog. 
+Logstash uses last checkpoint to resume processing the events in the backlog. 
```

Comment:
@colinsurprenant I am also with you on maybe not needing to discuss checkpoints in the way this document does today -- I think we should mention it with respect to durability and maybe nowhere else. At present, it's spread around the document and maybe needs to be just in one section.